### PR TITLE
Update to bootstrap 5

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
Upgrade to the latest UI kit version.
```

### CODE REVIEW

File: internal/admin/templates/bottom.html [hunk 1]
```diff
@@ -1,10 +1,8 @@
 {{define "bottom"}}
 </div>
 
-  <script src="https://code.jquery.com/jquery-3.5.1.min.js"
```
Comment (sethvargo) [inline internal/admin/templates/bottom.html:4]: Bootstrap doesn't need jquery anymore and I verified that we're not using it anywhere.

File: internal/admin/templates/top.html [hunk 2]
```diff
@@ -1,13 +1,20 @@
 {{define "top"}}
-<html>
+<html lang="en">
 <head>
   <title>{{if .title}}{{.title}}{{else}}Exposure Notifications{{end}}</title>
-  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
-    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
-  <link rel="stylesheet"
-    href="https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css"
```
Comment (sethvargo) [inline internal/admin/templates/top.html:8]: We used these icons exactly two places. It wasn't worth it. I removed it.

**Review by whaught — APPROVED** [review]
I assume you've loaded up these pages and they all look right

File: internal/admin/templates/bottom.html (hunk 1 above)
Comment (whaught) [reply internal/admin/templates/bottom.html:4]: this is awesome. Should load faster too!

**Review by mikehelmick — APPROVED** [review]

**Comment by sethvargo** [comment]
@whaught yea - I clicked around and did all the things

**Pull request merged**